bool enable_report_angles

